18:41:22.429 [main] ERROR com.eu.habbo.habbohotel.permissions.PermissionsManager - Caught SQL exception
java.sql.SQLException: java.lang.ArrayIndexOutOfBoundsException: Index 265 out of bounds for length 265
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:441)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1003)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery(ProxyPreparedStatement.java:52)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery(HikariProxyPreparedStatement.java)
	at com.eu.habbo.habbohotel.permissions.PermissionsManager.loadPermissions(PermissionsManager.java:44)
	at com.eu.habbo.habbohotel.permissions.PermissionsManager.reload(PermissionsManager.java:36)
	at com.furnibuilder.floorcommands.FloorCommands.checkDatabase(FloorCommands.java:84)
	at com.furnibuilder.floorcommands.FloorCommands.onEmulatorLoaded(FloorCommands.java:45)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.eu.habbo.plugin.PluginManager.fireEvent(PluginManager.java:333)
	at com.eu.habbo.Emulator.main(Emulator.java:167)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 265 out of bounds for length 265
	at com.mysql.cj.protocol.a.MergingColumnDefinitionFactory.createFromFields(MergingColumnDefinitionFactory.java:61)
	at com.mysql.cj.protocol.a.ColumnDefinitionReader.read(ColumnDefinitionReader.java:80)
	at com.mysql.cj.protocol.a.ColumnDefinitionReader.read(ColumnDefinitionReader.java:40)
	at com.mysql.cj.protocol.a.NativeProtocol.read(NativeProtocol.java:1583)
	at com.mysql.cj.protocol.a.BinaryResultsetReader.read(BinaryResultsetReader.java:70)
	at com.mysql.cj.protocol.a.BinaryResultsetReader.read(BinaryResultsetReader.java:50)
	at com.mysql.cj.protocol.a.NativeProtocol.read(NativeProtocol.java:1596)
	at com.mysql.cj.protocol.a.NativeProtocol.readAllResults(NativeProtocol.java:1650)
	at com.mysql.cj.ServerPreparedQuery.readExecuteResult(ServerPreparedQuery.java:389)
	at com.mysql.cj.ServerPreparedQuery.serverExecute(ServerPreparedQuery.java:208)
	at com.mysql.cj.jdbc.ServerPreparedStatement.serverExecute(ServerPreparedStatement.java:635)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:418)
	... 13 common frames omitted
18:43:27.821 [HabExec-1-2] ERROR com.eu.habbo.habbohotel.permissions.PermissionsManager - Caught SQL exception
java.sql.SQLException: java.lang.ArrayIndexOutOfBoundsException: Index 265 out of bounds for length 265
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:441)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1003)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery(ProxyPreparedStatement.java:52)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery(HikariProxyPreparedStatement.java)
	at com.eu.habbo.habbohotel.permissions.PermissionsManager.loadPermissions(PermissionsManager.java:44)
	at com.eu.habbo.habbohotel.permissions.PermissionsManager.reload(PermissionsManager.java:36)
	at com.eu.habbo.habbohotel.commands.UpdatePermissionsCommand.handle(UpdatePermissionsCommand.java:14)
	at com.eu.habbo.habbohotel.commands.CommandHandler.handleCommand(CommandHandler.java:88)
	at com.eu.habbo.habbohotel.rooms.Room.talk(Room.java:3172)
	at com.eu.habbo.habbohotel.rooms.Room.talk(Room.java:3099)
	at com.eu.habbo.messages.incoming.rooms.users.RoomUserTalkEvent.handle(RoomUserTalkEvent.java:33)
	at com.eu.habbo.messages.PacketManager.handlePacket(PacketManager.java:213)
	at com.eu.habbo.threading.runnables.ChannelReadHandler.run(ChannelReadHandler.java:24)
	at com.eu.habbo.threading.ThreadPooling.lambda$run$0(ThreadPooling.java:48)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 265 out of bounds for length 265
	at com.mysql.cj.protocol.a.MergingColumnDefinitionFactory.createFromFields(MergingColumnDefinitionFactory.java:61)
	at com.mysql.cj.protocol.a.ColumnDefinitionReader.read(ColumnDefinitionReader.java:80)
	at com.mysql.cj.protocol.a.ColumnDefinitionReader.read(ColumnDefinitionReader.java:40)
	at com.mysql.cj.protocol.a.NativeProtocol.read(NativeProtocol.java:1583)
	at com.mysql.cj.protocol.a.BinaryResultsetReader.read(BinaryResultsetReader.java:70)
	at com.mysql.cj.protocol.a.BinaryResultsetReader.read(BinaryResultsetReader.java:50)
	at com.mysql.cj.protocol.a.NativeProtocol.read(NativeProtocol.java:1596)
	at com.mysql.cj.protocol.a.NativeProtocol.readAllResults(NativeProtocol.java:1650)
	at com.mysql.cj.ServerPreparedQuery.readExecuteResult(ServerPreparedQuery.java:389)
	at com.mysql.cj.ServerPreparedQuery.serverExecute(ServerPreparedQuery.java:208)
	at com.mysql.cj.jdbc.ServerPreparedStatement.serverExecute(ServerPreparedStatement.java:635)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:418)
	... 20 common frames omitted
18:44:39.904 [HabExec-1-6] ERROR com.eu.habbo.habbohotel.permissions.PermissionsManager - Caught SQL exception
java.sql.SQLException: java.lang.ArrayIndexOutOfBoundsException: Index 265 out of bounds for length 265
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:441)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1003)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery(ProxyPreparedStatement.java:52)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery(HikariProxyPreparedStatement.java)
	at com.eu.habbo.habbohotel.permissions.PermissionsManager.loadPermissions(PermissionsManager.java:44)
	at com.eu.habbo.habbohotel.permissions.PermissionsManager.reload(PermissionsManager.java:36)
	at com.eu.habbo.habbohotel.commands.UpdatePermissionsCommand.handle(UpdatePermissionsCommand.java:14)
	at com.eu.habbo.habbohotel.commands.CommandHandler.handleCommand(CommandHandler.java:88)
	at com.eu.habbo.habbohotel.rooms.Room.talk(Room.java:3172)
	at com.eu.habbo.habbohotel.rooms.Room.talk(Room.java:3099)
	at com.eu.habbo.messages.incoming.rooms.users.RoomUserTalkEvent.handle(RoomUserTalkEvent.java:33)
	at com.eu.habbo.messages.PacketManager.handlePacket(PacketManager.java:213)
	at com.eu.habbo.threading.runnables.ChannelReadHandler.run(ChannelReadHandler.java:24)
	at com.eu.habbo.threading.ThreadPooling.lambda$run$0(ThreadPooling.java:48)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 265 out of bounds for length 265
	at com.mysql.cj.protocol.a.MergingColumnDefinitionFactory.createFromFields(MergingColumnDefinitionFactory.java:61)
	at com.mysql.cj.protocol.a.ColumnDefinitionReader.read(ColumnDefinitionReader.java:80)
	at com.mysql.cj.protocol.a.ColumnDefinitionReader.read(ColumnDefinitionReader.java:40)
	at com.mysql.cj.protocol.a.NativeProtocol.read(NativeProtocol.java:1583)
	at com.mysql.cj.protocol.a.BinaryResultsetReader.read(BinaryResultsetReader.java:70)
	at com.mysql.cj.protocol.a.BinaryResultsetReader.read(BinaryResultsetReader.java:50)
	at com.mysql.cj.protocol.a.NativeProtocol.read(NativeProtocol.java:1596)
	at com.mysql.cj.protocol.a.NativeProtocol.readAllResults(NativeProtocol.java:1650)
	at com.mysql.cj.ServerPreparedQuery.readExecuteResult(ServerPreparedQuery.java:389)
	at com.mysql.cj.ServerPreparedQuery.serverExecute(ServerPreparedQuery.java:208)
	at com.mysql.cj.jdbc.ServerPreparedStatement.serverExecute(ServerPreparedStatement.java:635)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:418)
	... 20 common frames omitted
18:44:53.389 [HabExec-1-2] ERROR com.eu.habbo.habbohotel.permissions.PermissionsManager - Caught SQL exception
java.sql.SQLException: java.lang.ArrayIndexOutOfBoundsException: Index 265 out of bounds for length 265
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:441)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1003)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery(ProxyPreparedStatement.java:52)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery(HikariProxyPreparedStatement.java)
	at com.eu.habbo.habbohotel.permissions.PermissionsManager.loadPermissions(PermissionsManager.java:44)
	at com.eu.habbo.habbohotel.permissions.PermissionsManager.reload(PermissionsManager.java:36)
	at com.eu.habbo.habbohotel.commands.UpdatePermissionsCommand.handle(UpdatePermissionsCommand.java:14)
	at com.eu.habbo.habbohotel.commands.CommandHandler.handleCommand(CommandHandler.java:88)
	at com.eu.habbo.habbohotel.rooms.Room.talk(Room.java:3172)
	at com.eu.habbo.habbohotel.rooms.Room.talk(Room.java:3099)
	at com.eu.habbo.messages.incoming.rooms.users.RoomUserTalkEvent.handle(RoomUserTalkEvent.java:33)
	at com.eu.habbo.messages.PacketManager.handlePacket(PacketManager.java:213)
	at com.eu.habbo.threading.runnables.ChannelReadHandler.run(ChannelReadHandler.java:24)
	at com.eu.habbo.threading.ThreadPooling.lambda$run$0(ThreadPooling.java:48)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 265 out of bounds for length 265
	at com.mysql.cj.protocol.a.MergingColumnDefinitionFactory.createFromFields(MergingColumnDefinitionFactory.java:61)
	at com.mysql.cj.protocol.a.ColumnDefinitionReader.read(ColumnDefinitionReader.java:80)
	at com.mysql.cj.protocol.a.ColumnDefinitionReader.read(ColumnDefinitionReader.java:40)
	at com.mysql.cj.protocol.a.NativeProtocol.read(NativeProtocol.java:1583)
	at com.mysql.cj.protocol.a.BinaryResultsetReader.read(BinaryResultsetReader.java:70)
	at com.mysql.cj.protocol.a.BinaryResultsetReader.read(BinaryResultsetReader.java:50)
	at com.mysql.cj.protocol.a.NativeProtocol.read(NativeProtocol.java:1596)
	at com.mysql.cj.protocol.a.NativeProtocol.readAllResults(NativeProtocol.java:1650)
	at com.mysql.cj.ServerPreparedQuery.readExecuteResult(ServerPreparedQuery.java:389)
	at com.mysql.cj.ServerPreparedQuery.serverExecute(ServerPreparedQuery.java:208)
	at com.mysql.cj.jdbc.ServerPreparedStatement.serverExecute(ServerPreparedStatement.java:635)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:418)
	... 20 common frames omitted
18:45:00.356 [HabExec-1-6] ERROR com.eu.habbo.habbohotel.permissions.PermissionsManager - Caught SQL exception
java.sql.SQLException: java.lang.ArrayIndexOutOfBoundsException: Index 265 out of bounds for length 265
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:441)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1003)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery(ProxyPreparedStatement.java:52)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery(HikariProxyPreparedStatement.java)
	at com.eu.habbo.habbohotel.permissions.PermissionsManager.loadPermissions(PermissionsManager.java:44)
	at com.eu.habbo.habbohotel.permissions.PermissionsManager.reload(PermissionsManager.java:36)
	at com.eu.habbo.habbohotel.commands.UpdatePermissionsCommand.handle(UpdatePermissionsCommand.java:14)
	at com.eu.habbo.habbohotel.commands.CommandHandler.handleCommand(CommandHandler.java:88)
	at com.eu.habbo.habbohotel.rooms.Room.talk(Room.java:3172)
	at com.eu.habbo.habbohotel.rooms.Room.talk(Room.java:3099)
	at com.eu.habbo.messages.incoming.rooms.users.RoomUserTalkEvent.handle(RoomUserTalkEvent.java:33)
	at com.eu.habbo.messages.PacketManager.handlePacket(PacketManager.java:213)
	at com.eu.habbo.threading.runnables.ChannelReadHandler.run(ChannelReadHandler.java:24)
	at com.eu.habbo.threading.ThreadPooling.lambda$run$0(ThreadPooling.java:48)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 265 out of bounds for length 265
	at com.mysql.cj.protocol.a.MergingColumnDefinitionFactory.createFromFields(MergingColumnDefinitionFactory.java:61)
	at com.mysql.cj.protocol.a.ColumnDefinitionReader.read(ColumnDefinitionReader.java:80)
	at com.mysql.cj.protocol.a.ColumnDefinitionReader.read(ColumnDefinitionReader.java:40)
	at com.mysql.cj.protocol.a.NativeProtocol.read(NativeProtocol.java:1583)
	at com.mysql.cj.protocol.a.BinaryResultsetReader.read(BinaryResultsetReader.java:70)
	at com.mysql.cj.protocol.a.BinaryResultsetReader.read(BinaryResultsetReader.java:50)
	at com.mysql.cj.protocol.a.NativeProtocol.read(NativeProtocol.java:1596)
	at com.mysql.cj.protocol.a.NativeProtocol.readAllResults(NativeProtocol.java:1650)
	at com.mysql.cj.ServerPreparedQuery.readExecuteResult(ServerPreparedQuery.java:389)
	at com.mysql.cj.ServerPreparedQuery.serverExecute(ServerPreparedQuery.java:208)
	at com.mysql.cj.jdbc.ServerPreparedStatement.serverExecute(ServerPreparedStatement.java:635)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:418)
	... 20 common frames omitted
21:35:18.893 [HabExec-1-3] ERROR com.eu.habbo.core.DatabaseLogger - Exception caught while saving loggables to database.
java.sql.BatchUpdateException: Data truncation: Incorrect string value: '\xE2\xAD\x90\xEF\xB8\x8F' for column `vicehabbo`.`chatlogs_room`.`message` at row 1
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.mysql.cj.util.Util.handleNewInstance(Util.java:192)
	at com.mysql.cj.util.Util.getInstance(Util.java:167)
	at com.mysql.cj.util.Util.getInstance(Util.java:174)
	at com.mysql.cj.jdbc.exceptions.SQLError.createBatchUpdateException(SQLError.java:224)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeBatchedInserts(ClientPreparedStatement.java:755)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeBatchInternal(ClientPreparedStatement.java:426)
	at com.mysql.cj.jdbc.StatementImpl.executeBatch(StatementImpl.java:796)
	at com.zaxxer.hikari.pool.ProxyStatement.executeBatch(ProxyStatement.java:128)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeBatch(HikariProxyPreparedStatement.java)
	at com.eu.habbo.core.DatabaseLogger.save(DatabaseLogger.java:37)
	at com.eu.habbo.core.CleanerThread.run(CleanerThread.java:85)
	at com.eu.habbo.threading.ThreadPooling.lambda$run$0(ThreadPooling.java:48)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Incorrect string value: '\xE2\xAD\x90\xEF\xB8\x8F' for column `vicehabbo`.`chatlogs_room`.`message` at row 1
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:104)
	at com.mysql.cj.jdbc.ServerPreparedStatement.serverExecute(ServerPreparedStatement.java:637)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:418)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1092)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1040)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1347)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeBatchedInserts(ClientPreparedStatement.java:716)
	... 14 common frames omitted
21:35:58.854 [HabExec-1-4] ERROR com.eu.habbo.core.DatabaseLogger - Exception caught while saving loggables to database.
java.sql.BatchUpdateException: Data truncation: Incorrect string value: '\xF0\x9F\xA4\xA9' for column `vicehabbo`.`chatlogs_room`.`message` at row 1
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.mysql.cj.util.Util.handleNewInstance(Util.java:192)
	at com.mysql.cj.util.Util.getInstance(Util.java:167)
	at com.mysql.cj.util.Util.getInstance(Util.java:174)
	at com.mysql.cj.jdbc.exceptions.SQLError.createBatchUpdateException(SQLError.java:224)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeBatchedInserts(ClientPreparedStatement.java:755)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeBatchInternal(ClientPreparedStatement.java:426)
	at com.mysql.cj.jdbc.StatementImpl.executeBatch(StatementImpl.java:796)
	at com.zaxxer.hikari.pool.ProxyStatement.executeBatch(ProxyStatement.java:128)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeBatch(HikariProxyPreparedStatement.java)
	at com.eu.habbo.core.DatabaseLogger.save(DatabaseLogger.java:37)
	at com.eu.habbo.core.CleanerThread.run(CleanerThread.java:85)
	at com.eu.habbo.threading.ThreadPooling.lambda$run$0(ThreadPooling.java:48)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Incorrect string value: '\xF0\x9F\xA4\xA9' for column `vicehabbo`.`chatlogs_room`.`message` at row 1
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:104)
	at com.mysql.cj.jdbc.ServerPreparedStatement.serverExecute(ServerPreparedStatement.java:637)
	at com.mysql.cj.jdbc.ServerPreparedStatement.executeInternal(ServerPreparedStatement.java:418)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1092)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1040)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1347)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeBatchedInserts(ClientPreparedStatement.java:716)
	... 14 common frames omitted
